pkg/app/pipectl/cmd/image/pushreference.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipe/pkg/app/api/service/apiservice"
+	"github.com/pipe-cd/pipe/pkg/cli"
+)
+
+type pushReference struct {
+	root *command
+
+	repoName string
+	tags     []string
+	digest   string
+}
+
+func newPushReferenceCommand(root *command) *cobra.Command {
+	c := &pushReference{
+		root: root,
+	}
+	cmd := &cobra.Command{
+		Use:   "push-reference",
+		Short: "Push a container image reference.",
+		RunE:  cli.WithContext(c.run),
+	}
+
+	cmd.Flags().StringVar(&c.repoName, "repo-name", c.repoName, "The repository name of container image. e.g. gcr.io/pipecd/pipecd, envoyproxy/envoy-alpine...")
+	cmd.Flags().StringSliceVar(&c.tags, "tag", c.tags, "The image tag.")
```

Comment:
It explicitly indicates a single tag. The reason why it doesn't allow the comma-separated list is that it's unneeded in the current use case?

Reply:
>  it doesn't allow the comma-separated list

It allows specifying the list of tags as `--tag=a --tag=b` or `--tag=a,b`.

The help is showing the type of that field as `strings` too.
The current description sentence is in favor of a single tag because I think that is more understandable for most of the users. But we can change that sentence. I have no strong opinion about that.

Reply:
Ok, in that case, looks good. Instead, the description should be more readable like:

```suggestion
	cmd.Flags().StringSliceVar(&c.tags, "tag", c.tags, "The image tag; Can be used multiple times to set multiple tags.")
```